ClosedXML/Excel/Ranges/XLRangeAddress.cs

```diff
@@ -123,7 +123,7 @@ IXLAddress IXLRangeAddress.LastAddress
             set { LastAddress = value as XLAddress; }
         }
 
-        public bool IsInvalid { get; set; }
+        public bool IsValid { get; set; } = true;
```

Comment:
Make setter internal?

Reply:
`XLRangeAddress` is already an internal class, so the setter is by default internal.

Reply:
My bad